resources/variants/agile_0.2.inst.cfg

```diff
@@ -0,0 +1,14 @@
+[general]
+name = 0.2 mm
+version = 4
+definition = agile
```

Comment:
The definition profile now is named as `zyyx_agile.def.json`. This needs to be changed to `zyyx_agile`.